Find a functionally equivalent code:android.view.HardwareRenderer.setupDiskCache:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * Sets the directory to use as a persistent storage for hardware rendering
 * resources.
 *
 * @param cacheDir A directory the current process can write to
 *
 * @hide
 */

@@ -1,3 +1,3 @@
 {
-    nSetupShadersDiskCache(new File(cacheDir, CACHE_PATH_SHADERS).getAbsolutePath());
+    ThreadedRenderer.setupShadersDiskCache(new File(cacheDir, CACHE_PATH_SHADERS).getAbsolutePath());
 }

